style.css

```diff
@@ -0,0 +1,1086 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+html {
+  scroll-behavior: smooth;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.desktop-head {
+  display: none;
+}
+
+.nav {
+  display: flex;
+  width: 100%;
+  height: 47px;
+  align-items: center;
+  padding-bottom: 40px;
+  padding-top: 20px;
+}
+
+.menu-btn {
+  padding-right: 40px;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px 20px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: black;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.top-links {
+  position: fixed;
+  left: -100%;
+  top: 40px;
+  gap: 0;
+  display: flex;
+  flex-direction: column;
+  background-color: rgba(0, 0, 0, 0.5);
+  width: 100%;
+  text-align: left;
+  transition: 0.5s;
+  height: 100em;
+  z-index: 999;
+  padding: 0 30px;
+  backdrop-filter: blur(7px);
+}
+
+.top-link {
+  margin: 16px 0;
+  text-decoration: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 44px;
+  color: white;
+}
+
+.top-links.active {
+  left: 0;
+}
+
+.main {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 10px;
+  padding-bottom: 70px;
+  background-image: url(../images/bgmobile.jpeg);
+  background-size: cover;
+}
+
+.main-bg {
+  position: absolute;
+  background: url("images/header-illsutration-mobile.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
```

Comment:
- [ ] I've noticed a minor issue in your design that's causing a **horizontal scrollbar** to appear on your page. 📜 
  To help you understand the problem better and explore potential solutions, I recommend watching the following video:
  
  [31_2023_1-10-48 PM.webm](https://github.com/mohamedgamaleldin999999/Capstone-project-module-one/assets/107952724/34f38dae-94ed-4492-a67e-77e2845118c0)
  
  _This video will provide valuable insights into the issue and offer potential solutions. Thank you for your attention to this matter! 🙏😊_